packages/core/src/shader/state/StencilState.ts

```
@@ -171,6 +179,31 @@
       writeMask
     } = this;
 
+    if (customStates) {
+      const enabledState = customStates[RenderStateElementKey.StencilStateEnabled];
+      enabledState !== undefined && (enabled = <boolean>enabledState);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)